src/common/window/page-visibility.js

```diff
@@ -10,8 +10,8 @@ import { documentAddEventListener } from '../event-listener/event-listener-opts'
  * @param {boolean} [toHiddenOnly=false] - only execute the 'cb' when the vis is changing to the hidden state; no arg is passed to 'cb' if used
  * @returns void
  */
-export function subscribeToVisibilityChange (cb, toHiddenOnly = false) {
-  documentAddEventListener('visibilitychange', handleVisibilityChange)
+export function subscribeToVisibilityChange (cb, toHiddenOnly = false, capture, abortSignal) {
```

Comment:
```suggestion
export function subscribeToVisibilityChange (cb, toHiddenOnly = false, capture = false, abortSignal = undefined) {
```
```suggestion
export function subscribeToVisibilityChange (cb, toHiddenOnly = false, capture, abortSignal) {
```
Prob better to turn into an object param though.

Reply:
Agree about the object param. Don't agree with defaulting to undefined, JS defaults args to undefined already.

Reply:
will kick the can, but should do the object params